### PR TITLE
create Db

### LINKED ISSUE #29: DB Creation
creat a DB for users:
email and username 
password
firstName
last_name
age
date 
profession

ccompanys:
UrlCompany 
CompanyName

and now we have super admin in the admin URL:  http://127.0.0.1:8000/admin/
user: ultimatejob
password: 1234

### PR DESCRIPTION
**Pull request by BarBenNaim**
fixes #29 

after sim nots:
I set up automatic import of models to the admin panel
I migrated between our users and users of django

### CODE REVIEW

**Comment by BarBenNaim** [comment]
I saw what flake8 wrote
but it's ok for me because I will use with timezone later

File: ultimatejobweb/admin.py [hunk 1]
```diff
@@ -1,3 +1,14 @@
-# from django.contrib import admin
+from django.contrib import admin
+from .models import user, company
+from django.apps import apps
 
 # Register your models here.
+admin.site.register(user)
+admin.site.register(company)
```
Comment (simzacks) [inline ultimatejobweb/admin.py:7]: why manually register user and company, if they'll get registered automatically in the code below?

**Review by simzacks — CHANGES_REQUESTED** [review]

File: ultimatejobweb/admin.py (hunk 1 above)
Comment (BarBenNaim) [reply ultimatejobweb/admin.py:7]: because they already exist. I have to so it that way.
I tried without it and it doesn't work

Comment (simzacks) [reply ultimatejobweb/admin.py:7]: The code below is supposed to register all the models including the ones above.
If that doesn't work, then there is no point in the code below at all.

**Pull request closed without merging**